YANG/ccamp/otn-tunnel/ietf-otn-tunnel.yang

```diff
@@ -532,40 +625,26 @@ module ietf-otn-tunnel {
     }
   }
 
-  /* Augment label hop of path-route of reverse primary path */
-  augment "/te:te/te:tunnels/te:tunnel/"
-        + "te:p2p-primary-paths/te:p2p-primary-path/"
-        + "te:p2p-primary-reverse-path/"
-        + "te:computed-paths-properties/te:computed-path-properties/"
-        + "te:path-properties/te:path-route-objects/"
-        + "te:path-computed-route-object/te:type/te:label/"
-        + "te:label-hop/te:te-label/te:technology" {
-    description "OTN label.";
-    case otn {
-      uses l1-types:otn-label-hop;
-    }
-  }
-
-  /* Augment label hop of record-route of reverse primary LSP */
+  /* Augment label-step of the label restrictions for the forwarding
+     direction of path-out-segment of primary reverse path */
   augment "/te:te/te:tunnels/te:tunnel/"
-        + "te:p2p-primary-paths/te:p2p-primary-path/"
-        + "te:p2p-primary-reverse-path/"
-        + "te:lsps/te:lsp/te:lsp-record-route-information/"
-        + "te:lsp-record-route-information/te:type/te:label/"
-        + "te:label-hop/te:te-label/te:technology" {
+        + "te:primary-paths/te:primary-path/"
+        + "te:primary-reverse-path/"
+        + "te:path-out-segment/te:label-restrictions/"
+        + "te:label-restriction/te:label-step/te:technology" {
```

Comment:
same comment, why no augmentation for label-start and label end?

Reply:
see comment above: clarified that the label-start and label-end augmentations are already defined. This PR is adding also the augmentations for the label-step.